cms/tests/test_toolbar.py

```diff
@@ -200,6 +200,18 @@ def test_toolbar_request_endpoint_validation(self):
                 },
             )
             self.assertEqual(response.status_code, 200)
+            self.assertContains(response, "Clipboard")
+
+            response = self.client.get(
+                endpoint,
+                data={
+                    'obj_id': page_content.pk,
+                    'obj_type': 'cms.pagecontent',
+                    'cms_path': get_object_edit_url(page_content)+"q"  # Invalid
+                },
+            )
+            self.assertEqual(response.status_code, 200)
+            self.assertNotContains(response, "Clipboard")  # No clipboard exposed to invalid cms_path
```

Comment:
```suggestion
            # No clipboard exposed to invalid cms_path
            self.assertNotContains(response, "Clipboard")
```

It's not great having comments appended at the end of code that is not say settings, they are easily missed and having them on their own line helps to keep it consistent with the rest of the codebase.